### ISSUE TITLE
Name and Device reference number fields are missing in sharing device

### ISSUE THREAD

**Issue by Anushree09-N** [mosip/inji#225]
**Describe the bug**
Name and Device reference number fields are missing in sharing device

**To Reproduce**
Device A --> Requesting
Device B --> Sharing

Open qrcode in Device A
Open scanner in Device B
Scan the qrcode from Device B
connection is established, Now select VC to share screen appears.

**Current behavior**
Name and Device reference number fields are missing in sharing device , We see only **requested by** and **reason for sharing fields**.

**Expected behavior**
Sharing Device screen should contain Name and Device reference number field.

**Screenshots**
![image](https://user-images.githubusercontent.com/79500509/200549026-93b92466-ce07-43f7-b343-5391232fc68d.png)

**Smartphone (please complete the following information):**
Device: iPhone7 & iPhone XR
OS: 15.4.1
Inji app version: 0.4.0 (16) in test flight
MOSIP Version: 1.2.1
MOSIP server: qa-121.mosip.net

**Comment by rakhimosip** [comment]
@kyanthony were these fields removed intentionally? If yes, then we need to ensure that there is no information loss post this change.

**Comment by pmigueld** [comment]
@rakhimosip Yes, the fields were removed [here](https://github.com/idpass/inji/commit/4a57e519375fd0e75645a68d38098a3706bced87#diff-900f23ae5b0715a51d80162db8aa5a257a0cee39c6fe11e05fd78ecd0c2402ceL15-L20). It was discussed before that it could be confusing to have multiple device identifiers on the screen. 

The ones removed were:
- **Name** - this was the user-configurable identifier from the settings page (e.g. Paolo's iPhone)
- **Device ID** - from this function call: [`RNDeviceInfo.getDeviceId()`](https://github.com/react-native-device-info/react-native-device-info#getdeviceid)

The one left:
- **Device Name** - this uses the name that is set on the user's device typically found in the bluetooth settings

**Comment by rakhimosip** [comment]
Thanks for clarifying @pmigueld 

**Question:** I don't remember setting the name in Bluetooth settings on my device but I do see a name I had used for a demo. So does this mean that Bluetooth uses the device name from phone settings the very first time?

@Anushree09-N @santhoshsunder please update the related test case as per @pmigueld comment above.

**Comment by pmigueld** [comment]
@rakhimosip The default name is usually set by the device manufacturer (like the model of the phone) but if it is not set it will be a unique string of letters that identifies it to other bluetooth devices.

**Comment by Anushree09-N** [comment]
**Verified on:** 
Device: iPhone 8 & iPhone 11
OS: 16.0.2 (20A380), 15.6.1
Inji app version: 0.4.1 (6) -> Testflight
MOSIP Version: 1.2.0.1
MOSIP server: qa5.mosip.net

**Observation:**  
Device A Requesting (iOS)
Device B Sharing (iOS)

**Prerequisite:** set Name in profile section on both the device 
**EX**:  
Device A ->Name>> Anushree1
Device B -> Name>> Hamsa1

![image](https://user-images.githubusercontent.com/79500509/208844946-ebcd0fc2-d151-4940-81b7-8b35fd4ae74b.png)

![image](https://user-images.githubusercontent.com/79500509/208845106-a9aba48a-4748-4ddc-9a8d-fca39eb35c11.png)

**Initiate the VC share from Device B to Device A** 

**Sharing Screen:** 

![image](https://user-images.githubusercontent.com/79500509/208845626-ba6bfbfc-bd02-4b42-b49a-befe6cca04c1.png)

**Requesting screen:** 

![image](https://user-images.githubusercontent.com/79500509/208845733-82ae1a81-d27e-4da2-8169-8dd0f9c8e333.png)

**Success popup on both device after VC share.**

**Sharing device:**

![image](https://user-images.githubusercontent.com/79500509/208845991-6d097ca4-4451-40e5-81f6-575f55cf233c.png)

**Requesting device:**

![image](https://user-images.githubusercontent.com/79500509/208846074-5f5dd862-f077-40aa-b32d-c1c4e7cc54f6.png)

**History for VC share for both device**

**Shared device:**

![image](https://user-images.githubusercontent.com/79500509/208848066-c94646bc-8a27-4c65-8d03-7f7f3ee9877d.png)

**Requesting device:**

![image](https://user-images.githubusercontent.com/79500509/208848123-3541aff9-f5e0-4a84-b896-848627d54ca5.png)

Name is set on both device under profile section and while sharing we see the device name as set in bluetooth in phone settings but in history tab we see the name which is set in the profile section. Attached the reference screenshots above . 

CC @rakhimosip @pmigueld
